src/components/QuakeByMagnitude/MagnitudeMap.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import L from 'leaflet';
+import '../../../node_modules/leaflet/dist/leaflet.css';
+
+function MagnitudeMap({ earthquakes }) {
+  useEffect(() => {
+    const mapContainer = document.getElementById('map');
```

Comment:
* As mentioned in the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Home%20page,the%20details%20page.),
 ```
The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.
Along with the category name, you will display some numeric values per category that come from the API.
 When the user clicks (or taps) on a category item, the application navigates to the details page. 
``` Currently, your Home page doesn't have all these interactions, kindly follow the project requirements